lib/keycloak.js

```diff
@@ -9,6 +9,7 @@ module.exports = (app) => {
   if (KEYCLOAK_CONFIG) {
     log.info('KEYCLOAK_CONFIG was detected. Enabling keycloak protections.')
     log.debug('KEYCLOAK_CONFIG was set to: %j', KEYCLOAK_CONFIG)
+    KEYCLOAK_CONFIG['auth-server-url'] = KEYCLOAK_CONFIG['auth-server-url'].replace(/\/$/, "")
```

Comment:
Would be worth adding a comment explaining this.

Reply:
Good point. Done that now.